modules/dxp/apps/analytics/analytics-reports-web/src/main/java/com/liferay/analytics/reports/web/internal/data/provider/AnalyticsReportsDataProvider.java

```diff
@@ -332,16 +321,17 @@ public Map<TrafficChannel.Type, TrafficChannel> getTrafficChannels(
 			Collection<AcquisitionChannel> values =
 				acquisitionChannels.values();
 
-			Stream<AcquisitionChannel> stream = values.stream();
+			List<TrafficChannel> trafficChannelList = TransformUtil.transform(
+				values, TrafficChannel::newInstance);
+
+			Map<TrafficChannel.Type, TrafficChannel> trafficChannelMap =
```

Comment:
@slnn Please name this map `trafficChannels`. The list variable above is not needed, please inline it.